src/test/analyzersilence_test.cpp

```diff
@@ -212,4 +213,25 @@ TEST_F(AnalyzerSilenceTest, RespectUserEdits) {
     EXPECT_DOUBLE_EQ(kManualOutroPosition.value(), pOutroCue->getLengthFrames());
 }
 
+TEST_F(AnalyzerSilenceTest, verifyFirstSound) {
+    std::array<const CSAMPLE, 14> s{0.0000f,
```

Comment:
two options here if we don't want to specify the length explicitly
let CTAD do the work (and don't forget the include)
```suggestion
    std::array s{0.0000f,
```
Or fallback to a plain C array (which is fine IMO when passed to a span right afterwards)
```suggestion
    const CSAMPLE s[] = {0.0000f,
```